Makefile

```diff
@@ -17,7 +17,10 @@ include machine_settings.make
 raw_data=gmted2010_modis
 
 #intermediate_cubed_sphere_file=/project/amp/juliob/topo-data/gmted2010_modis-ncube3000-stitch.nc
-intermediate_cubed_sphere_file=/glade/p/cgd/amp/aherring/grids/topo/gmted2010_modis-ncube3000-stitch.nc
+#intermediate_cubed_sphere_file=/glade/p/cgd/amp/aherring/grids/topo/gmted2010_modis-ncube3000-stitch.nc
+intermediate_cubed_sphere_file=/shared/wmr/mcurry/cesmdata/topo/gmted2010_modis-ncube3000-stitch.nc
+intermediate_cubed_sphere_file=/shared/wmr/mcurry/cesmdata/topo/gmted2010_bedmachine-ncube6000.nc
+
```

Comment:
assuming these are on MMM clusters?

Reply:
Yes! These are all available on MMM machines, currently, whitedwarf, eris and orcus as well as to user computers that are connected to the internet while in the building.